CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ of response and not the raw API endpoint response
 - **CUMULUS-2644**
   - Pulled `delete` method for `granules-executions.ts` implemented as part of CUMULUS-2306
   from the RDS-Phase-2 feature branch in support of CUMULUS-2644.
+  - Changed `erasePostgresTables` in serve.js to ensure granules_executions, granules, pdrs, are
```

Comment:
Can you update the entry here to what you do for this PR?

Reply:
Absolutely. Updated and pushed. Thanks!